examples/running_flct.py

```diff
@@ -0,0 +1,122 @@
+"""
+===================================
+Fourier Local Correlation Tracking
+===================================
+
+This example applies Fourier Local Correlation Tracking (FLCT)
+to a set of two images taken within a short interval of each other
+using `~sunkit_image.flct.flct`.
+"""
+
+import numpy as np
+
+from sunkit_image.flct.flct import flct
+import matplotlib.pyplot as plt
+
+###########################################################################
+# This examples demonstrates how to find the
```

Comment:
```suggestion
# This examples demonstrates how to find the 2D velocity flow field.
```

Reply:
But for this to be true, you will have to construct a flow field at the end of the example.